telearm/imu_fusion.py

```
@@ -0,0 +1,480 @@
+    Create mock IMU fusion data for testing.
+    
+    Args:
+        num_samples: Number of pose samples to generate
+        
+    Returns:
+        List of mock operator poses
+    """
+    estimator = OperatorPoseEstimator()
+    poses = []
+    
+    for i in range(num_samples):
```

Ruff (0.14.0)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)